app/assets/stylesheets/building_map.scss

```diff
@@ -53,6 +53,52 @@
   text-align: center;
 }
 
+.pin-icon1 {
```

Comment:
```suggestion
.pin-icon1
.pin-icon2 {
```
because those two classes are just the same.

Reply:
Yes, we are aware, that this is code duplication, but we tried different approaches and this was the only one we could find, that worked. If you manage to find a concrete way of changing this, which works, we would be happy to implement it :)

Reply:
I changed this